src/Theme/ClassicTheme.php

```diff
@@ -46,15 +47,21 @@ public function onTestFinished(TestResult $result): void
             $throwable = $throwable->previous();
         }
 
-        if ($result->trace && !$result->trace->suppressed) {
-            printf(
-                Color::colorize("fg-$statusColour", '%s%s: %s in %s on line %s%1$s%1$s'),
-                PHP_EOL,
-                $result->status->name,
-                $result->trace->message,
-                $result->trace->file,
-                $result->trace->line,
-            );
+        $firstIssue = true;
+        foreach ($result->uniqueTraces as $trace) {
+            if (!$trace->suppressed) {
+                $issueStatusColour = self::getColour($trace->issueStatus);
```

Comment:
Since this is a new feature involving colours, can we have some screenshots attached to this PR for posterity?

Reply:
Added to description.